iree/compiler/Dialect/VM/Target/CMakeLists.txt

```diff
@@ -14,12 +14,19 @@
 
 iree_add_all_subdirs()
 
+if(IREE_ENABLE_EMITC)
```

Comment:
To make the `bazel_to_cmake` check happy,

Add `bazel_to_cmake: DO NOT EDIT`
https://github.com/google/iree/blob/3a87a04f18a87a68a68fcb6cb1a2f3db7abc2c9e/build_tools/bazel_to_cmake/bazel_to_cmake.py#L40-L42

like https://github.com/google/iree/blob/ef26761e7b6788bd7484b3ac53ea62dc45cd68e6/iree/samples/vulkan/CMakeLists.txt#L27

OR `.skip_bazel_to_cmake` file 

https://github.com/google/iree/blob/3a87a04f18a87a68a68fcb6cb1a2f3db7abc2c9e/build_tools/bazel_to_cmake/bazel_to_cmake.py#L106

like https://github.com/google/iree/blob/main/bindings/python/pyiree/common/.skip_bazel_to_cmake

Reply:
Done :) I was aware of the `# bazel_to_cmake` line, but was wondering if there is a preferred mechanism and who `iree/tool/CMakeLists.txt` is skipped.